improver/calibration/ensemble_calibration.py

```diff
@@ -1098,6 +1058,7 @@ def process(self, current_forecast, coefficients_cube, landsea_mask=None):
         """
         self.current_forecast = current_forecast
         self.coefficients_cube = coefficients_cube
+        self.landsea_mask = landsea_mask
```

Comment:
I don't think this line is required because the `landsea_mask` is only used within the `process` method of the plugin.